### DISCUSSION TITLE
[Unrelated] Multiple glyph entry in IBus?

### DISCUSSION THREAD

**Discussion by gregdan3** [dec32/ajemi#9] [General]
Hi; you don't have a discussion section open, so I'm opening an issue to ask: Do you happen to know if there's a way to do multiple glyph entry in IBus? This input mode that Ajemi seems to default to is significantly faster than even the suggestions I made in #2, but I'm usually on Linux to Ajemi isn't available to me.

**Comment by dec32** [comment]
Not sure what you mean by "multiple glyph entry" but maybe press <kbd>Ctrl</kbd> + <kbd>/</kbd> to toggle off auto-commit mode if you are using ibus-table.

**Reply by gregdan3** [reply]
Apologies for the delay in response! I forgot to pick this back up.

Turning off auto-commit mode only solves this part-way. 
With auto commit, I enter a word as soon as I match it exclusively, or commit manually. Without it, I can continue typing, but I no longer have any matches as soon as I begin typing a second word. I cannot match if I type two words consecutively. 
In Ajemi, if I type two words consecutively, the match engine is clever enough to see that the string matches two consecutive words- in fact, it does this no matter the size of the input string.

This is the fastest way I've seen to do SP input- it just lets me keep typing and commit whenever my sentence is complete, assuming I don't make any typoes along the way. But Ajemi is only on windows, aha.
